internal/cli/daemon.go

```
@@ -86,8 +113,14 @@ func init() {
 
 func runStart(cmd *cobra.Command, _ []string) error {
 	d := daemon.New(config.DefaultDataPath())
+	cfg, cfgErr := loadDaemonConfig(cmd)
```

golangci-lint (2.12.2)

[error] ineffectual assignment to cfgErr

(ineffassign)